src/account-type-utils.js

```diff
@@ -0,0 +1,45 @@
+const _ = require('lodash')
+const accountTypeMapping = require('../config/account-types.json')
+
+/**
+ * Maps the email domain to an account type.
+ * To add or remove types, please refer to the ../config/account-types.json file.
+ * The key is the account type, and the value is the mapped domains.
+ *
+ * @param {string} email The email address to map.
+ * @returns {string} Returns the account type related to the domain.
+ * @example
+ *
+ * getAccountType('gdematto@broadinstitute.org')
+ * // => 'Broad Employee'
+ *
+ * getAccountType('email@gmail.com')
+ * // => 'Independent User'
+ *
+ * getAccountType('email@mit.edu')
+ * // => 'Educational Institution User'
+ *
+ */
+
+const getAccountType = email => {
+  if (!_.includes(email, '@') ||
```

Comment:
this should work. I believe the [hapi/joi](https://joi.dev/api/?v=17.6.0#stringemailoptions) library (that this service uses) has some email validation that you could use.

I have not specifically used the email validation from this library, and don't want to hold up the PR on this.